lib/flok.js

```diff
@@ -149,19 +150,13 @@ Flok.prototype.up = function up(opts, done) {
       self._lock(callback);
     },
 
-    migrations: ['lock', function (callback) {
-      self._loadMigrations(callback);
-    }],
-
-    migrationStatus: ['migrations', function (callback) {
-      self._loadMigrationsStatus(callback);
+    loadMigrations: ['lock', function (callback) {
+      self.load(callback);
     }],
 
-    sortMigrations: ['migrationStatus', function (callback) {
-      self.sortMigrationsUp(callback);
-    }],
+    incompatibleMigrations: ['loadMigrations', (callback) => this.checkV1Migrations(callback)],
```

Comment:
It's not related to changes in this PR, but probably it's good to refactor usage of `self`.